cves/kernel/CVE-2019-19069.yml

```diff
@@ -84,14 +87,9 @@ fixes_instructions: |
 
   Place any notes you would like to make in the notes field.
 fixes:
-- commit:
-  note:
-- commit:
-  note:
 - commit: fc739a058d99c9297ef6bfd923b809d85855b9a9
   note: |
-    Taken from NVD references list with Git commit. If you are
-    curating, please fact-check that this commit fixes the vulnerability and replace this comment with 'Manually confirmed'
+    Manually confirmed
```

Comment:
I would suggest to add what was added and deleted to the code in the commit for more clarity in this section, but this is not required just a suggestion